components/admin_console/group_settings/groups_list/groups_list.tsx

```diff
@@ -453,14 +485,16 @@ export default class GroupsList extends React.PureComponent {
                         </div>
                         <button
                             className={'btn btn-link prev ' + (firstPage ? 'disabled' : '')}
-                            onClick={firstPage ? null : this.previousPage}
+                            // eslint-disable-next-line no-undefined
+                            onClick={firstPage ? undefined : this.previousPage}
```

Comment:
Not sure about this, you are already handling the button being diabled and previouspage is safe enough as it will redirect you to the same page, so I wouldn't make any comparisons here and setting it as `undefined` as it doesn't gain anything out of it and might trigger an error if something tries to call it.
Also it will allow us to remove the eslint exception on the comment.

Reply:
Thanks for the review, These changes require an update to jest snapshots. Okay to do it ?

Reply:
sure, I don't think there is a problem on updating snapshots if required.